lib/data/landing_page_content_items/homepage.yaml

```diff
@@ -53,8 +53,8 @@ blocks:
   featured_content:
     blocks:
       - type: heading
-        content: Our tasks
-        path: /landing-page/tasks
+        content: Our progress
+        path: /landing-page/goals
```

Comment:
I think this is the wrong page, the design shows it linking to the tasks page.

Reply:
Ah okay thanks will check.

Reply:
Changed it back, sorry, think that was a rebase error.